packages/ui/src/users/UsersList.tsx

```diff
@@ -121,13 +125,33 @@ const UsersList: FunctionComponent = () => {
 
   const generateDefaultUser = (): UserWithOrg => {
     return {
-      ...(new UserWithOrg()),
+      ...new UserWithOrg(),
       permissions: [
         PERMISSIONS.CAN_MANAGE_DOCUMENTS,
         PERMISSIONS.CAN_VIEW_DOCUMENTS,
-      ]
+      ],
+    };
+  };
+
+  const onUserDelete = async (user: UserWithOrg) => {
+    try {
+      setState({
+        loadingMessage: 'Deleting user',
+      });
+      await httpClient.user.delete(user);
+      await loadData();
+    } catch (e) {
+      notification.alert({
+        type: NOTIFICATION.ERROR,
+        title: 'Failed to delete user',
+        message: (e as AxiosError)!.response!.data.message,
```

Comment:
I tihnk you should be able to remove the first `!` here, since you're casting it as `AxiosError` so it's already not undefined, right? 

```suggestion
        message: (e as AxiosError).response!.data.message,
```